docs/accessories/general/slot_types.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Creating and Modifying Slots
+project: accessories
+---
+
+If such information below is still leaving you with questions, you can look at each platform's test mod [[Fabric](https://github.com/wisp-forest/accessories/tree/fa06f044f5c7486b26a8c0774f7ca3edbd256cad/fabric/src/testmod), [Neoforge](https://github.com/wisp-forest/accessories/tree/fa06f044f5c7486b26a8c0774f7ca3edbd256cad/neoforge/src/testmod)] for starting points. You can also ask questions within the Discord {todo: add discord link} for more assistance.
+
+Creating a slot within Accessories requires you either follow the Datapack method which allows for full ability to tweak Slot Properties and its bound Entities. 
+
+???+ tip "Notice of Datapackablity"
+    It is recommend that when taking this approach that you design your Mod in mind that it may allow for certain behaviors to be moved from one slot to another, remove your target slot, or even prevent a item from being equipped in any slot.
```

Comment:
This sentence seems grammatically correct, but I don't actually get what its trying to say...

Reply:
Made adjustments to hopfully make more sense... I hope